src/main/target/KAKUTEF4/target.h

```diff
@@ -87,11 +87,9 @@
 #define USE_I2C_DEVICE_1
 #define I2C_DEVICE              (I2CDEV_1)
 
-#if defined(FLYWOOF405)
-#define I2C1_SCL                PB6
-#else 
+
```

Comment:
This extra blank line is bogus. There is another down below. Tidiness.

Reply:
Let me rewrite it. Sorry

Reply:
Don't file a new PR.

Make corrections and push them.

When the PR look good, you will going to _SQUASH_ commits.
If you don't know how to do it, we will tell you how.

Reply:
Ok, please teach me merging ，so thank ...